runtime/near-vm-runner-standalone/src/script.rs

```diff
@@ -118,22 +119,26 @@ impl Script {
         let config_store = RuntimeConfigStore::new(None);
         let runtime_fees_config = &config_store.get_config(self.protocol_version).transaction_costs;
         let mut outcomes = Vec::new();
-        for step in &self.steps {
-            for _ in 0..step.repeat {
-                let res = run_vm(
-                    &self.contracts[step.contract.0],
-                    &step.method,
-                    &mut external,
-                    step.vm_context.clone(),
-                    &self.vm_config,
-                    runtime_fees_config,
-                    &step.promise_results,
-                    self.vm_kind,
-                    self.protocol_version,
-                    self.contract_cache.as_deref(),
-                );
-                outcomes.push(res);
+        if let Some(runtime) = self.vm_kind.runtime() {
```

Comment:
Nit: As we go through both branches anyway, maybe use match?

Reply:
Yeah, the TODO below indicates that I'm not particularly happy with the approach here. Will keep this nit in mind when I'm working on this TODO.